docs/docs/deploy/heroku.md

```diff
@@ -0,0 +1,170 @@
+
+And we're set! This means that the next time we merge into `master` we'll see our application automatically updated!
+
+### Step 4: Making a change
+
+Let's make a tiny change. I've added a new empty line between two tags in `src/webapp/src/templates/pages/index.peb`. You could do something more dramatic here of course.
+
+Commit this change on a feature branch `feature/example` and push to github. Now go to your repo in the browser and you'll 
+be prompted to open a pull-request. Click the `Compare & pull request...` button, fill out the form on this page and then 
+create the pull request. 
+
+[![github pull request with CI running](/assets/images/heroku-deploy/github-pull-request-ci-running.png)](/assets/images/heroku-deploy/github-pull-request-ci-running.png)
+
+Alright! Now go make a cup of coffee while the CI build runs. Springboard has a longer build than a base Spring Boot project, 
+but we promise it's worth it. Right now GitHub is running unit, integration and feature tests on the back-end and has 
+kicked off Cypress to test your front-end. Once it's done you should see something like this:
+
+[![pull request with deployment](/assets/images/heroku-deploy/github-pull-request-review-created.png)](/assets/images/heroku-deploy/github-pull-request-review-created.png)
+
+Click the `View dpeloyment` button and... Ta-da!
+
+[![pull request with deployment](/assets/images/heroku-deploy/review-app-deployed.png)](/assets/images/heroku-deploy/review-app-deployed.png)
+
+Our review app is live! This has it's own Heroku Postgres instance spun up so we can play around with it without impacting a shared database.
+
+### Step 5: Merging to master and deploying to production
+
+Now that we've done some QA on our review app, we'll merge the pull request and deploy to production. Normally, this is 
+where a teammate would review our change, but for now we can safely click the `Merge` button and carry on solo.
+
+Upon merging the pull request, the review app instance will be spun down - saving us some money/free dyno hours. At this point you can head back to the heroku pipeline to view the deploy logs.
+
+[![heroku deployment in progress](/assets/images/heroku-deploy/heroku-deploy-logs.png)](/assets/images/heroku-deploy/heroku-deploy-logs.png)
+
+Give it a couple of minutes to run and your app will be deployed! You can also check in on the deployment in the GitHub environment screen, by clicking the `environments` button on your repo's home page.
+
+[![github env is pending](/assets/images/heroku-deploy/github-env-pending.png)](/assets/images/heroku-deploy/github-env-pending.png)
+
+Once the app has been deployed this will change to read `Deployed`. 
+
+Click the `View deployment` button... and it's broken! This is because Heroku doesn't respect the `app.json` file's addons in production. I don't fully understand why they made this choice, but let's fix it.
```

Comment:
Which part of the app.json. ... what addons. I guess I'm confused as to what you expect with the app.json and what it actually does?